test/nfkd-compat-test.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+
+'use strict';
+
+const assert = require('bsert');
+const nfkd = require('../lib/hd/nfkd-compat');
+
+describe('NFKD-Compat', function() {
+  it('should apply compatibility normalization as expected', () => {
+    // unicode char ﬀ (U+FB00) is normalized to ff (U+0066 U+0066)
+    assert.equal(nfkd('\uFB00').localeCompare('\u0066\u0066'), 0);
+  });
+
+  it('should apply compatibility norm. when nfkd is passed a string if normalize function is undefined', () => {
```

Comment:
Same as last PR, please squash your commits and try to rephrase this test title so it fits in 80 columns. Great contribution!

Reply:
Done!